airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/integrations.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
```

Comment:
Use `number` as type instead of `string`

Reply:
Similar to other ids, it's a UUID. Sample API response:

```
{
    "data": [
        {
            "id": "00c3f61c-7f8a-45f5-8789-3896d1e36ae1",
            "name": "Product_Analytics_Prometheus",
            "enabled": true,
            "type": "Prometheus",
            "teamId": "34fcb8f5-564f-4093-befe-efc7d29b7ff1",
            "version": "v2"
        },
        ...
    ],
    "took": 0.011,
    "requestId": "07f539e4-27b0-4754-9b05-3f51cf463cd4"
}
```